flocker/ca/test/test_ca.py

```diff
@@ -341,6 +381,21 @@ def test_certificate_subject_control_service(self):
         subject = cert.get_subject()
         self.assertEqual(subject.CN, b"control-service")
 
+    def test_subjectAltName(self):
+        """
+        The written certificate has a subjectAltName containing the given
+        hostname.
+        """
+        assert_has_extension(self, self.credential.credential,
+                             b"subjectAltName", b"DNS:control.example.com")
+
+    def test_extendedKeyUsage(self):
+        """
+        The written certificate has extendedKeyUsage set to "serverAuth".
```

Comment:
These two test method docstrings duplicate the prevailing convention in this test suite of saying the assertion is about a certificate that is "written" somewhere but the tests just use an in-memory object.